pilot/pkg/bootstrap/server.go

```diff
@@ -747,18 +751,15 @@ func (s *Server) mcpController(args *PilotArgs,
 	clients *[]*sink.Client,
 	configStores *[]model.ConfigStoreCache) {
 	clientNodeID := ""
-	var collections []sink.CollectionOptions
+	collections := make([]sink.CollectionOptions, len(schemas.Istio)-1)
```

Comment:
this is not right, this will result in empty indexes at the start

Reply:
Yes, you are right, and I made a change